intro-HPC/ch_running_jobs_with_input_output_data.tex

```diff
@@ -364,6 +364,14 @@ \subsection{Your UGent home drive and shares \label{subsec:ugent-shares}}
 
 Each hour the process will check if your ticket should be renewed.
 
+
+We advise to disable access to your shares once access is no longer needed:
+
+\begin{prompt}
+%\shellcmd{kdestroy}%
```

Comment:
I would use `kdestroy -A`

Reply:
i thought about that, but it will also destroy any other tokens (however unlikely)
i would only destroyt the tokens created (if possible).

Reply:
So which token will `kdestroy` destroy then? Does it ask, or what?

Reply:
ALL OF THEM.

Reply:
you can either:
 - destroy all caches of the user (-A)
 - destroy a specific cache (-c)
 - destroy the default cache (no flags)

in practice, I doubt a user will use anything else than the default cache location.

Reply:
ah, it destroys the whole cache, not individual tokens.
it looks like `klist purge` is someting one could use for that, but too complicated to explain and not worth it. `kdestroy` is more than enough